src/chapters/setup-intro-data-types/data-types.rst

```diff
@@ -84,95 +81,80 @@ are a few examples of popular languages falling into these two categories:
-the most common data types in this language.
+Because we need to give plenty of attention to types when writing Java code, let’s begin 
+by exploring the most common data types in this language.
+
+Strings and Single Characters
+-----------------------------
+
+Immutability
+^^^^^^^^^^^^
 
-String
--------
+Strings in Java are immutable. This means a string cannot be changed; A given character 
+inside a string in cannot be changed. 
 
-Strings in Java and JavaScript are quite similar. Like JavaScript, Java strings
-are immutable. However, unlike JavaScript, there IS a difference between using
-single and double quotes around strings.
+Single vs. Double Quotation Marks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. admonition:: Examples
+Java syntax requires double quotation marks when declaring strings. 
 
-   The following JavaScript declarations are all valid:
+Java has another variable type, ``char``, which is used for a single character. ``char`` 
+uses single quotation marks. The single character can be a letter, digit, punctuation, 
+or whitespace like tab (``'\t'``).
 
-   #. ``let str = 'LaunchCode'``;
-   #. ``let str = "LaunchCode"``;
-   #. ``let str = 'A'``;
-   #. ``let str = "A"``;
+.. sourcecode:: java
+   :linenos:
 
-   In Java, single quotes are used around the single character data type
-   ``char``. Double quotes enclose String data types:
+   String staticVariable = "dog";
+   char charVariable = 'd';
 
-   #. ``char charVariable = 'a'``;
-   #. ``String stringVariable = "LaunchCode"``;
 
-   The single character can be a letter, digit, punctuation, or whitespace like
-   tab (``'\t'``).
+Manipulation
+^^^^^^^^^^^^
 
-Manipulating strings in Java looks similar to our experience with JavaScript,
-but with a slightly different syntax. The table below summarizes the
-differences between the string methods available for the two languages. For the
-examples shown in the table, we will use the string variable
+The table below summarizes some of the most common string methods available in Java.
+the two languages. For the examples shown in the table, we use the string variable
 ``String str = "Rutabaga"``.
 
-.. list-table:: String manipulations in JavaScript vs. Java
+.. list-table:: String manipulations in Java
    :header-rows: 1
 
-   * - JavaScript
-     - Java
+   * - Java Syntax
      - Description
-   * - ``str[3]``
-     - ``str.charAt(3)``
+   * - ``str.charAt(3)``
      - Returns the character at index 3, (``'a'``).
-   * - ``str[2:5]``
-     - ``str.substring(2,4)``
+   * - ``str.substring(2,4)``
      - Returns the characters from indexes 2 - 4, (``"tab"``).
-   * - ``str.length``
-     - ``str.length()``
+   * - ``str.length()``
      - Returns the length of the string.
    * - ``str.indexOf('a')``
-     - ``str.indexOf('a')``
      - Returns the index for the first occurrence of 'a', (``3``).
-   * - ``str.split('delimiter')``
-     - ``str.split('delimiter')``
+   * - ``str.split(<delimiter>)``
```

Comment:
``delimiter`` is enclosed with double quotes, not <>.  ``str.split("delimiter")``.